evm-contracts/test/v0.7/Operator.test.ts

```diff
@@ -296,7 +194,7 @@ describe('Operator', () => {
         })
 
         it('emits an event', async () => {
-          assert.equal(receipt.events?.length, 1)
+          assert.equal(receipt.events?.length, 2)
```

Comment:
assert on type of second event?

Reply:
Good point, the extra event is from the mock that I added. I'll add this in.

Reply:
So, both `Operator` and `OperatorForwarder` emit the `AuthorizedSendersChanged` event now?

This seems redundant. Can we save gas by having only one of these emit the event?

Reply:
We should really emit an event every time the state changes on a contract. Since both contracts have their state changed, we should emit that.

In this scenario, the `Operator` contract has 3 authorized senders in its state, whereas the `OperatorForwarder` has 4 (the original 3, plus the `owner` of the `Operator`). So there is a difference that we should capture in emitting these events.

We could use better naming for them to distinguish between the two? `ForwarderAuthorizedSendersChanged` for example?

Reply:
So, the owner is always an authorized sender on the `OperatorForwarder` by design? I think its ok to have the same name of the event if they're coming from different contracts.